content/en/docs/ops/deployment/requirements/index.md

```diff
@@ -60,7 +88,7 @@ requirements:
 
 ## Ports used by Istio
 
-The following ports and protocols are used by Istio.
+Applications should avoid the following ports and protocols, which are used by Istio.
```

Comment:
you do not need to avoid the ones in Istiod. just the envoy ones. Otherwise people would be pretty mad we reserved 443 and 8080 :slightly_smiling_face:

Reply:
Good point :)

It does beg the question of why we need to document the non-envoy ones anyway.

Reply:
It seems reasonable to put it <somewhere> but not here. not sure we have a doc where it does make sense, either. I would be ok with removal